core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -258,30 +260,93 @@ var TopicEditorPage = function() {
       'Create subtopic modal taking too long to disappear.');
   };
 
-  this.dragSkillToSubtopic = async function(skillIndex) {
-    var target = element(by.css('.protractor-test-subtopic-column'));
-    await waitFor.visibilityOf(target,
-      "Subtopic's skills list taking too long to appear.");
-    var toMove = await skillCards.get(skillIndex);
+  this.dragSkillToSubtopic = async function(skillDescription, subtopicIndex) {
+    await waitFor.visibilityOf(await uncategorizedSkills.first(),
```

Comment:
I don't think you need to `await` the `.first()` call since `.first()` returns an ElementFinder, not a promise

Reply:
Done.